forge/db/models/Device.js

```diff
@@ -38,10 +38,14 @@ module.exports = {
     hooks: function (M, app) {
         return {
             beforeCreate: async (device, options) => {
-                const deviceLimit = app.license.get('devices')
-                const deviceCount = await M.Device.count()
-                if (deviceCount >= deviceLimit) {
-                    throw new Error('license limit reached')
+                // if the product is licensed, we permit overage
+                const isLicensed = app.license.active()
+                if (isLicensed !== true) {
```

Comment:
There's no code-path that returns anything but a bool, I think.

```suggestion
                if (app.license.active()) {
```

Reply:
Likely I'm just not aware of JS things in this comment, completely fine to ignore me. :)

Reply:
it is for readability mostly.  `app.license.active()` does not scream "is licensed" to me.

Reply:
also, your suggestion inverted the logic but not the operations (would break if committed).
Will skip this suggestion ZJ. but thanks for reviewing.